src/App/Resources/en-US/Resources.resw

```diff
@@ -172,7 +172,7 @@
   <data name="BadIpContent" xml:space="preserve">
     <value>Could not connect to {0}.
 
-Check that the IP address is correct and that the Factory Orchestrator Service is running on the target IP.</value>
+Check that the IP address, Server Name and Certificate Hash  is correct and that the Factory Orchestrator Service is running on the target IP.</value>

```

Comment:
while i was lazy, can you please add a comment since we are changing this string?